Makefile

```diff
@@ -49,8 +48,15 @@ unit-test-wasm: depend
 	@scripts/check_unit_wasm.sh
 
 .PHONY: bdd-test
-bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker
-	@scripts/check_integration.sh
+bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker bdd-js bdd-go
```

Comment:
note: this will require local installation of chrome to run bdd-test target on a developer's system.

Reply:
https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/bdd_tests.md#run-all-tests

Reply:
No - puppeteer downloads headless chrome during installation